pkg/kwokctl/components/kwok_controller.go

```diff
@@ -73,6 +78,34 @@ func BuildKwokControllerComponent(conf BuildKwokControllerComponentConfig) (comp
 				ReadOnly:  true,
 			},
 		)
+
+		objs, err := config.Load(context.Background(), conf.ConfigPath)
```

Comment:
```suggestion
		logs := config.FilterWithTypeFromContext[*internalversion.Logs](ctx)
		clusterLogs := config.FilterWithTypeFromContext[*internalversion.ClusterLogs](ctx)
```

Reply:
I like this, and we can leverage `runtime.GetLogVolumes` to avoid logic replication. Then only thing is that we don't have a context here. Can I change the function signature to accept context?

Reply:
It seems that this should be moved outside of `BuildKwokControllerComponent` so that the unit tests for `BuildKwokControllerComponent` have no dependencies, if any.

Reply:
That is even better